docker/packager/packager

```diff
@@ -149,6 +149,7 @@ def parse_env_variables(
     PPC_SUFFIX = "-ppc64le"
     RISCV_SUFFIX = "-riscv64"
     S390X_SUFFIX = "-s390x"
+    LOONGARCH_SUFFIX = "-loongarch64"
```

Comment:
It looks as if more changes are required in this file to enable CI builds for LoogArch64. Please see https://github.com/ClickHouse/ClickHouse/pull/63135/files#diff-00c66281b1b2973ae73eab7b260f1ef5ecc0a72ce4cec1b8fa71922d31c1f9dd for a related PR (about s390x).